VideoLocker/res/values/content_descriptions.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!-- TalkBack mode pronounces "edX" as "Ed". This forces the annunciation. -->
+    <string name="content_description_edx">Ed-X</string>
```

Comment:
@mdinino have you tested this with the string Ed X?  I worry that on some devices, or even different user settings (users have the ability to control the level of punctuation that is announced by Talkback) the name could be pronounced as Ed dash X.